.golangci.yml

```diff
@@ -0,0 +1,33 @@
+version: "2"
+run:
+  # disallows implicit automatic updating of go.mod and fails when any changes to go.mod are needed
+  modules-download-mode: readonly
+
+linters:
+  enable:
```

Comment:
I know it is not in the scope of this PR, but it would be great to have the same (or as similar as possible) configuration among all repositories with Go.